src/dxvk/dxvk_memory.cpp

```diff
@@ -34,7 +34,7 @@ namespace dxvk {
   DxvkMemory& DxvkMemory::operator = (DxvkMemory&& other) {
     m_chunk   = std::exchange(other.m_chunk,  nullptr);
     m_heap    = std::exchange(other.m_heap,   nullptr);
-    m_memory  = std::exchange(other.m_memory, VkDeviceMemory(nullptr));
+    m_memory  = std::exchange(other.m_memory, VkDeviceMemory(0));
```

Comment:
this should be `VK_NULL_HANDLE` instead of 0 or `nullptr`.

Reply:
Settng the C++ version to c++17 will break GCC 6.x, which is what currently ships with Ubuntu. Breaking that is questionable, but I'm not sure if there is a way to set different values for different compilers.